.github/workflows/main.yml

```diff
@@ -18,6 +18,14 @@ jobs:
     name: Build
     uses: ./.github/workflows/_build.yml
 
+  ReleaseDraft:
```

Comment:
drafting the release notes as a separate job to ensure it still gets drafted even if the other workflows on main fail

Reply:
what's the purpose of creating the release notes if the other workflows fail?

Reply:
This runs on main after a PR gets merged. Right not CI is a bit flaky due to the android emulator, but we always want to make sure the release notes get drafted even if main fails for some reason